src/main/scala/cromwell/engine/db/slick/LocalJobComponent.scala

```diff
@@ -1,5 +1,6 @@
 package cromwell.engine.db.slick
 
+//TODO: Maybe replace the name of this to just, Job, say?
```

Comment:
Yeah, it's the only thing which seems to make sense - I just worry about that table getting ridiculously big.

I suppose if it *really* becomes a problem we could hide those queries behind some other actor which is talking to a DB which is more optimized for this type of thing